src/shared/src/types/change-request-types.ts

```diff
@@ -81,6 +82,13 @@ export enum ChangeRequestReason {
   Other = 'OTHER'
 }
 
+export enum ChangeRequestStatus {
+  Implemented = 'IMPLEMENTED',
```

Comment:
can we make all of these like `Implemented` `Accepted`

(i know all the others are all caps but it's actually so annoying that they're all caps becuase we have to make them not all caps all the time when we display them lol so this gets us ahead of that)